standalone/src/main/resources/cz/crcs/ectester/standalone/libs/jni/Makefile

```diff
@@ -112,21 +113,18 @@ openssl.o: openssl.c
 # BoringSSL shim
 boringssl: boringssl_provider.so
 
-lib_boringssl.so:
-	cp $(PROJECT_ROOT_PATH)/ext/boringssl/build/crypto/libcrypto.so lib_boringssl.so
-
-boringssl_provider.so: boringssl.o c_utils.o | clibs lib_boringssl.so
-	$(CC) $(LFLAGS) -o $@ -Wl,-rpath,'$$ORIGIN/lib' $^ -L. lib_boringssl.so -l:lib_timing.so -l:lib_csignals.so
+boringssl_provider.so: boringssl.o c_utils.o | clibs
+	$(CC) $(CFLAGS) $(LFLAGS) -o $@ -Wl,-rpath,'$$ORIGIN/lib' $^ -L. -Wl,-Bstatic -l:lib_boringssl.a -Wl,-Bdynamic -l:lib_timing.so -l:lib_csignals.so
 
 boringssl.o: boringssl.c
-	$(CC) -I$(PROJECT_ROOT_PATH)/ext/boringssl/include/ $(CFLAGS) -c $<
+	$(CC) -I$(BORINGSSL_CFLAGS) $(CFLAGS) -c $<
```

Comment:
The CFLAGS should not go directly after the `-I`. Rather, include the `-I` in the cflags.